pom.xml

```diff
@@ -79,21 +81,39 @@
   </distributionManagement>
 
   <properties>
-    <checkstyle.version>6.1.1</checkstyle.version>
+    <maven.compiler.source>1.6</maven.compiler.source>
+    <maven.compiler.target>1.6</maven.compiler.target>
+
+    <version.com.puppycrawl.tools.checkstyle>6.1.1</version.com.puppycrawl.tools.checkstyle>
+    <version.com.mycila.license-maven-plugin>2.7</version.com.mycila.license-maven-plugin>
+    <version.org.eclipse.jgit>3.6.1.201501031845-r</version.org.eclipse.jgit>
+    <version.junit>4.12</version.junit>
   </properties>
```

Comment:
Minor comment

AFAIC, I don't see the value in extracting all versions to properties (I do see value when the version number is used twice or more)

Is this a rule for POM writing we're supposed to obey or only your own practice?